auth_token/utils.py

```diff
@@ -416,7 +429,8 @@ def create_otp(slug, related_objects=None, data=None, key_generator=None, expira
     if deactivate_old:
         deactivate_otp(slug, related_objects=related_objects)
 
-    key_generator = import_string(settings.OTP_DEFAULT_KEY_GENERATOR) if key_generator is None else key_generator
+    key_generator = settings.OTP_DEFAULT_KEY_GENERATOR if key_generator is None else key_generator
```

Comment:
Shouldn't here be used `AUTHORIZATION_OTP_BACKEND_DEFAULT_KEY_GENERATOR` ?

Reply:
No this is create_otp function not authorization backend. There realy need to be default OTP generator. Maybe you didn't open hidden code